pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -182,6 +183,14 @@ function validateParams(vmParams) {
         }
     }
 
+    if (vmParams.unattendedInstallation && !vmParams.rootPassword) {
+        validationFailed.rootPassword = _("Please insert a valid root password");
```

Comment:
"insert" sounds a bit strange here IMHO -- maybe "set"?

Also, we shouldn't require both a root and a user password (sudo!), one ought to be enough? E. g. Ubuntu doesn't use root passwords at all by default in its installers and all released media -- you can set it of course, but it's highly discouraged.

Reply:
So I think I misunderstood the man pages of virt-install

The parameters where admin-password-file and user-password-file
I renamed the UI parameters to reflect that - s/Root/Admin/

I also made the user not required - if not provided only root account will exist (not accepted in Debian workstation - I just tried it - but accepted in Fedora 30)
If admin password will be passed it will be the root password and also the sudo password for the new user I think.
I will verify it right now.

Reply:
You are right. I adjusted the code accordingly.